src/data/security-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const securityQuiz = [
+  {
+    Question: "Where is the password of a Linux user stored?",
+    Answer: "/etc/shadow",
+    Distractor1: "/etc/passwd",
+    Distractor2: "/etc/shadow",
+    Distractor3: "/bin/passwd",
+    Explanation: "The password of a Linux user is stored in /etc/shadow for security reasons.",
+    Link: "https://www.freecodecamp.org/news/how-to-manage-users-in-linux/"
+  },
```

Comment:
I am asking for clarity. Is it a practice here to check all questions against outside resources? (I am asking to be aware of  all sites/resources I need to compare my questions against, otherwise, it will be difficult to come up with questions).

Reply:
> Is it a practice here to check all questions against outside resources?

Yes, it is important that you come up with your own questions and answers. Otherwise, it will look to much like plagiarism and we don't want freeCodeCamp to be blamed for that.

In this situation here, your question and combination of answers were the issue. 

If it were me, here is how I would have rephrased the questions and answers

```md
 {
    Question: "In Linux, where is a user's password kept?",
    Answer: "/etc/shadow",
    Distractor1: "/bin",
    Distractor2: "/private/password",
    Distractor3: "/user/root/bin/password",
    Explanation: "In Linux, hashed passwords are stored in the shadow file (/etc/shadow). Hashing refers to the process of transforming the original password into a string of characters that appear in a random order. Hashed passwords are used for security reasons and only the root user has access to read or modify the shadow file.",
    Link: "https://www.freecodecamp.org/news/how-to-manage-users-in-linux/"
  },
```

> otherwise, it will be difficult to come up with questions

Yes, it can be difficult to come up with questions. But remember it is not the question by itself we are flagging. 

For example, questions like `What does the background-color property do in CSS?` or `What does the mkdir command do in Linux`, are not original. You can find those on dozens of quizzes. 

But your questions, combination of answers, and explanation need to be unique. So when you come up with your own combination like I just did, then you won't have to worry about it being to similar to what is already out there 👍